docs/md/docs/2020-11-02-turn-the-senior-around.md

```diff
@@ -0,0 +1,68 @@
+# Turn The Senior Around
+
+As engineering managers we take many considerations into account, two of the main ones are:
+- Delivering on time
+- Make our people grow
+
+During my experience as an engineering manager and as an engineer all the managers I have met saw those subjects critical and invested a lot in improving them.<br>
+Another consensus among them all was that senior engineers are a key enabler to both topics, they are the main players the team lead can rely on to achieve those goals.<br>
```

Comment:
A couple of suggestions:
- Break this sentence into two shorter sentences for better readability
- Replace `topics` with `responsibilities` to reflect that it is also a responsibility of every team member

`Another consensus among them all was that senior engineers are a key enabler to both responsibilities. They are the main players the team lead can rely on to achieve those goals.`